compiler/rustc_parse/src/parser/mod.rs

```diff
@@ -544,6 +546,11 @@ impl<'a> Parser<'a> {
         self.check_or_expected(self.token.can_begin_const_arg(), TokenType::Const)
     }
 
+    fn check_inline_const(&mut self) -> bool {
```

Comment:
This is more like `eat_inline_const` (or rather `eat_const_in_const_block`), functions that consume tokens are not normally named `check_*`.